generative_model_training_all_labels/04_generative_model_gig_all_labels.ipynb

```diff
@@ -0,0 +1,1764 @@
+{
```

Comment:
<p>Is there a reason to have the different edge types analyzed in different notebooks? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/snorkeling-full-text/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I couldn't decide which system is best. Either its going to be one notebook and I keep track of a whole lot of files or mini-changes or multiple notebooks that I can tune if there are special cases per edge type. I chose to do the latter, but could easily do the other way as well.